astropy/units/tests/test_format.py

```diff
@@ -471,6 +472,17 @@ def test_cds_non_ascii_unit():
         u.radian.find_equivalent_units(include_prefix_units=True)
 
 
+@pytest.mark.parametrize(
+    "string, unit",
+    [
+        ("eps0/mu0", u.cds.eps0 / u.cds.mu0),
+        ("a0.s", u.cds.a0 * u.cds.s),
+    ],
+)
+def test_cds_special_units_composite(string, unit):
+    assert u.Unit(string, format="cds") == unit
```

Comment:
Why are these two units tested here instead of [`test_cds_grammar()`](https://github.com/astropy/astropy/blob/3fde46588c6c8e06521f286a746c4298ff0fe544/astropy/units/tests/test_format.py#L129)?

Reply:
Yes, much better. Somehow I started with thinking of this as a regression, but really, it's just part of the grammar (even if a bit odd since one cannot do powers...).